ludwig/data/preprocessing.py

```diff
@@ -1305,6 +1305,14 @@ def build_preprocessing_parameters(
     return feature_name_to_preprocessing_parameters
 
 
+def is_input_feature(feature_config: FeatureConfigDict) -> bool:
+    """Utility function to check for the presence of encoder in the feature config to determine if the feature is
+    an input feature or output feature."""
+    if ENCODER in feature_config:
```

Comment:
is this always present on the input feature?

Reply:
@jppgks I believe it is by this point because we only reach preprocessing after the config object is created and validated